lib/fizzy/parser_expr.cpp

```diff
@@ -173,41 +175,74 @@ void push_branch_immediates(
     push(immediates, arity);
 }
 
-inline void mark_frame_unreachable(ControlFrame& frame) noexcept
+inline void mark_frame_unreachable(ControlFrame& frame, Stack<ValType>& operand_stack) noexcept
 {
     frame.unreachable = true;
-    frame.stack_height = frame.parent_stack_height;
+    operand_stack.shrink(static_cast<size_t>(frame.parent_stack_height));
 }
 
+inline void drop_operand(
```

Comment:
In the "Validation Algorithm" this is a `pop_opd()`. Is the returned type ever needed?

Reply:
I can rename it to `pop_operand`.

Though it's not exactly the same thing as `pop_opd()` in Validation Algorithm, where it's a low-level primitive used to build all other checks. There return value is needed to check against the expected value and also return value can be `Unknown`. 
We never push unknown values into stack, so here return value would always be equal to `expected_type` input value.

Reply:
You don't have to rename it. I was just curious.